q2_feature_classifier/classifier.py

```diff
@@ -233,7 +233,8 @@ def classify_sklearn(reads: DNAFASTAFormat, classifier: Pipeline,
 
 
 _classify_parameters = {
-    'reads_per_batch': Int % Range(0, None),
+    'reads_per_batch': Int % Range(0, None, inclusive_start=False) | Str % Choices(
```

Comment:
why not simplify to `Range(1, None) | Str % Choices(['auto'])`?

Reply:
I tried that, and I also tried changing the default from 0 to 1 or 'auto' or None. They all still produced the same error.